src/telemetry/class-parsely-a8c-tracks-event.php

```diff
@@ -0,0 +1,135 @@
+<?php
+/**
+ * Parsely_A8c_Tracks_Event class
+ *
+ * @package Parsely
+ * @since 3.0.0
+ */
+
+declare(strict_types=1);
+
+/**
+ * Instances of this class are fit for recording to the Automattic Tracks system (unless an error occurs during instantiation).
+ */
+class Parsely_A8c_Tracks_Event {
```

Comment:
Let's use a `Parsely\Telemetry` namespace with an unprefixed class name like `A8c_Tracks_Event` (and similar), for these new classes.